src/domain/entities/alert/AlertData.ts

```diff
@@ -0,0 +1,23 @@
+import { Maybe } from "../../../utils/ts-utils";
+import { Id } from "../Ref";
+
+export type OutbreakType = "disease" | "hazard";
+
+export type Alert = {
```

Comment:
you are exporting two types of Alert , one here and one in previous file - this is a clear code smell.